rust/src/domains/ffi.rs

```diff
@@ -500,10 +499,9 @@ pub extern "C" fn opendp_domains__user_domain(
     let identifier = try_!(to_str(identifier)).to_string();
     let descriptor = try_as_ref!(descriptor).clone();
     let element = ExtrinsicElement::new(identifier, descriptor);
-    let member = try_as_ref!(member).clone();
+    let member = wrap_func(try_as_ref!(member).clone());
     let member = Function::new_fallible(move |arg: &ExtrinsicObject| -> Fallible<bool> {
-        let c_res = (member.callback)(AnyObject::new_raw(arg.clone()));
-        Fallible::from(util::into_owned(c_res)?)?.downcast::<bool>()
+        member(&AnyObject::new(arg.clone()))?.downcast::<bool>()
```

Comment:
This is superficial, but with the redefinition of `member` it's trickier to understand the type. I would have trouble explaining whether `callback` is needed or not.